server/sonar-server/src/main/java/org/sonar/server/ruby/PlatformRubyBridge.java

```diff
@@ -42,44 +42,29 @@ public PlatformRubyBridge(RackBridge rackBridge) {
 
   @Override
   public RubyDatabaseMigration databaseMigration() {
-    final CallDatabaseVersionUpgrade callDatabaseVersionUpgrade = parseMethodScriptToInterface(
+    CallDatabaseVersionUpgrade callDatabaseVersionUpgrade = parseMethodScriptToInterface(
       CALL_UPGRADE_AND_START_RB_FILENAME, CallDatabaseVersionUpgrade.class
       );
 
-    return new RubyDatabaseMigration() {
-      @Override
-      public void trigger() {
-        callDatabaseVersionUpgrade.callUpgrade();
-      }
-    };
+    return callDatabaseVersionUpgrade::callUpgrade;
   }
 
   @Override
   public RubyRailsRoutes railsRoutes() {
-    final CallLoadJavaWebServices callLoadJavaWebServices = parseMethodScriptToInterface(
+    CallLoadJavaWebServices callLoadJavaWebServices = parseMethodScriptToInterface(
       CALL_LOAD_JAVA_WEB_SERVICES_RB_FILENAME, CallLoadJavaWebServices.class
       );
 
-    return new RubyRailsRoutes() {
-      @Override
-      public void recreate() {
-        callLoadJavaWebServices.callLoadJavaWebServices();
-      }
-    };
+    return callLoadJavaWebServices::callLoadJavaWebServices;
   }
 
   @Override
   public RubyMetricCache metricCache() {
-    final CallInvalidateMetricCache callInvalidateMetricCache = parseMethodScriptToInterface(
+    CallInvalidateMetricCache callInvalidateMetricCache = parseMethodScriptToInterface(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "callInvalidateMetricCache". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1854)